content/en/docs/concepts/workloads/controllers/_index.md

```diff
@@ -1,5 +1,58 @@
 ---
 title: "Workload Resources"
 weight: 20
+simple_list: true
 ---
 
+Kubernetes provides several built-in APIs for declarative management of your
+{{< glossary_tooltip text="workloads" term_id="workload" >}}
+and the components of those workloads.
+
+Ultimately, your applications run as containers inside
+{{< glossary_tooltip term_id="Pod" text="Pods" >}}; however, managing individual
+Pods would be a lot of effort. For example, if a `Pod` fails, you probably want to
+run a new Pod to replace it. Kubernetes can do that for you.
+
+You use Kubernetes API to create a workload
+{{< glossary_tooltip text="object" term_id="object" >}} that represent a higher level
+abstraction than a `Pod`, and then the Kubernetes
+{{< glossary_tooltip text="control plane" term_id="control-plane" >}} automatically manages
+Pod objects on your behalf, based on the specification for the workload object you defined.
+
+The built-in APIs for managing workloads are:
+
+[`Deployment`](/docs/concepts/workloads/controllers/deployment/) (and, indirectly, [`ReplicaSet`](/docs/concepts/workloads/controllers/replicaset/)),
+the most common way to run an application on your cluster.
+A Deployment is a good fit for managing a stateless application workload on your cluster:
+_stateless_ means that any {{< glossary_tooltip term_id="Pod" >}} in the Deployment is interchangeable,
+and can be replaced if needed.
```

Comment:
I'm not sure this definition is accurate.

Reply:
How's the revised text?